docs/docs/communication_between_robots.md

```diff
@@ -38,38 +38,30 @@ this channel are going to receive it. So, if `robot1` sends a message,
 
 ### Converting the message into a packet
 
-Imagine we would like to send values `v1`, `v2`, ... , `vn`.
-Before the message is sent to the channel, it must be converted into a packet.
-Simply put, the packet is just a bytes object representing values `v1`, `v2`, ... , `vn`.
-To convert our values into bytes object we can use built-in library
-called [struct](https://docs.python.org/3/library/struct.html), which is
-shipped together with Python.
-
-We need to know the type of each value we want to pack into a packet. Let's see a
-quick example. The values we want to send are `v1 = 3.14`, `v2 = 5` and `v3 = True`.
-It is obvious that `v1` is floating point number, `v2` is integer and `v3`
-is boolean. Knowing the type of each variable, we need to define the
-structure of the message.
+Webots supports sending the packet in different formats (string, bytes, list of doubles...).
+Since our example controllers as well as supervisor use strings, let's describe how to send it.
+
+[JSON](https://en.wikipedia.org/wiki/JSON) is a format that stores human-readable text in key-value pairs.
```

Comment:
@Adman For a while I was wondering whether using something like [`bson`](https://bsonspec.org/) wouldn't be a better fit for the embedded usecase but seeing how the SoccerSim gets targeted mostly at lowering the barrier to entry, JSON does seem preferable indeed.

Reply:
This is the first time I hear about bson.
I would stick with the json for its simplicity and secondly that it's builtin python3, so no third party dependencies get involved that you have to maintain.

Reply:
Sure, makes sense, it just was the first thing that crossed my mind when reading about this :slightly_smiling_face: